gr-utils/python/modtool/cli/add.py

```diff
@@ -107,7 +108,8 @@ def get_copyrightholder(self):
         with SequenceCompleter(copyright_candidates):
             self.info['copyrightholder'] = cli_input("Please specify the copyright holder: ")
             if not self.info['copyrightholder'] or self.info['copyrightholder'].isspace():
-                self.info['copyrightholder'] = "gr-{} author".format(self.info['modname'])
+                inf_=self.info['modname']
```

Comment:
this is *really* a nitpick, but while you're at it: Could you make it `variable = rvalue` instead of `variable=rvalue`?